src/backend-c/runtime/obj.h

```diff
@@ -0,0 +1,32 @@
+#ifndef OBJ_INCLUDED
```

Comment:
I pulled that file out from `runtime.h` so I could use it in my headers without importing the whole `runtime.h`. I also need to import `cast_queue.h` in `runtime.h` for declaring the global queues.